### ISSUE TITLE
Cookie size related exceptions in magento

### ISSUE THREAD

**Issue by sksksksk** [VivaPayments/API#2081]
I get critical exceptions logged in var/log/exception.log of my magento 2.3.6 installation from the viva payments gateway. It complains about the cookie size. For example

[2020-12-29 07:11:20] main.CRITICAL: Unable to send the cookie. Size of 'mage-messages' is 4155 bytes. {"report_id":"c8a8da5a120c1a9f05602531c5002a86a9318950b60e74a83c81b8e7b04b8ef8","exception":"[object] (Magento\\Framework\\Stdlib\\Cookie\\CookieSizeLimitReachedException(code: 0): Unable to send the cookie. Size of 'mage-messages' is 4155 bytes. at MY_PATH/app/code/Ced/VivaPayments/Stdlib/Cookie/CookieManager.php:263)"} []

The max size of the cookie is limited by the MAX_COOKIE_SIZE constant (in CookieManager.php), which is set to 4096. The 4096 size in the RFC is not strict, so this could be easily increased. The RC says "...At least 4096 bytes per cookie..." and not "at most".

**Comment by matthewrgourd** [comment]
Hi @sksksksk, I'm afraid I've not come across that issue before. Do please try to increase the cookie size in case that makes any difference. Let me know how you get on.

**Comment by matthewrgourd** [comment]
Hi @sksksksk, did that have the desired effect? Please close this issue if ok now...

**Comment by sksksksk** [comment]
sorry for not getting back to you. The shop owner had implemented a temporary workaround, and until he undoes it, I can't really test it.

What I can tell is that the error messages stopped after I increased the cookie size. I just don't know yet if the payments will be processed correctly from now on, and will know as soon as the shop owner has time to undo this temporary workaround.

And upon taking a closer look at these exceptions, I noticed that they occurred every ~28 minutes. So I assume that these belonged to requests from VIVA, and kept being sent as they always resulted in these exceptions.
